crypto/digestible/Cargo.toml

```diff
@@ -24,8 +24,8 @@ x25519-dalek = { version = "2.0.0-pre.2", default-features = false, optional = t
 [features]
 default = ["alloc", "derive", "dalek", "mc-crypto-dalek/default"]
 # Enables support for types in alloc crate
-alloc = ["mc-crypto-dalek/alloc"]
+alloc = []
 # Enables re-export of derive(Digestible) macro
 derive = ["mc-crypto-digestible-derive"]
 # Enables support for some crypto primitives in dalek crates
-dalek = ["mc-crypto-dalek", "curve25519-dalek", "ed25519-dalek", "x25519-dalek"]
+dalek = ["curve25519-dalek", "ed25519-dalek", "x25519-dalek"]
```

Comment:
For future reference, I'd also suggest not mixing/matching multiple things in the same PR, e.g. fiddling with all this can/should be it's own PR, aside from moving tx computation to a separate crate. That way, the separation can continue, and the dalek mess can get cleaned up independently.

Reply:
I mean, OK, that's a bug in `mc-crypto-dalek`, though it sounds like something that's worth fixing after we've unforked `curve25519-dalek` to 4.0.0-rc.1. Any reason not to just prioritize that work ahead of this, then fix this to handle it?